addons/editor/functions/fnc_handleLoad.sqf

```diff
@@ -128,10 +128,19 @@ GVAR(iconsVisible) = true;
 
 [{
     // For compatibility with Zeus Game Master missions, wait until the respawn placement phase is complete
+    // and the create trees have been refreshed after curator addons are changed
     [{
         params ["_display"];
 
-        isNull _display || {entities "ModuleMPTypeGameMaster_F" isEqualTo []} || {GETMVAR(BIS_moduleMPTypeGameMaster_init,false)}
+        isNull _display
+        || {entities "ModuleMPTypeGameMaster_F" isEqualTo []}
+        || {
+            GETMVAR(BIS_moduleMPTypeGameMaster_init,false)
+            && {
+                (_display displayCtrl IDC_RSCDISPLAYCURATOR_CREATE_MODULES) tvCount [] > 1
+                || {"CuratorModeratorRights" call BIS_fnc_getParamValue < 1 && {player != BIS_curatorUnit}}
```

Comment:
Checking for an empty module tree (_i.e._ `tvCount` returns `0`) should already be enough for moderators without being too mission specific.

Reply:
This line is for when moderators have no addons and can only spectate or edit existing objects. In that case, the tree is always empty.

Reply:
Nvm, leave it as it is. My suggestion doesn't even work.